MusicNotationKit/MusicNotationKit/Types.swift

```diff
@@ -25,22 +25,22 @@ public enum Striking {
 	case Right, Down
 }
 
-public enum NoteDuration {
-	case Whole
-	case Half
-	case Quarter
-	case Eighth
-	case Sixteenth
-	case ThirtySecond
-	case SixtyFourth
+public enum NoteDuration: String {
```

Comment:
The raw value here seems to be useful only for debugging purposes and may not be the raw value we want for actual development purposes. So instead of tying the raw value to a `String` now instead of either a `Double` or no raw value, let's use the same debug string protocol and a switch statement to return these strings.